src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/index.jsx

```diff
@@ -14,37 +14,42 @@ import getEmbedUrl from '#lib/utilities/getEmbedUrl';
 
 const staticAssetsPath = `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}`;
 
-const liveRadioPlaceholderImageSrc = `${staticAssetsPath}images/amp_audio_placeholder.png`;
+const audioPlaceholderImageSrc = `${staticAssetsPath}images/amp_audio_placeholder.png`;
 
-const MediaPlayerOuterWrapper = styled.div`
+const getMediaInfo = assetId => ({
+  title: assetId === 'liveradio' ? 'Live radio' : 'On-demand radio',
+  type: 'audio',
+});
+
+const getMasterBrand = (masterBrand, liveRadioIdOverrides) =>
+  pathOr(masterBrand, ['masterBrand', masterBrand], liveRadioIdOverrides);
+
+const OuterWrapper = styled.div`
   @media (min-width: 63rem) {
     display: flex;
     justify-content: center;
   }
 `;
 
-const MediaPlayerInnerWrapper = styled.div`
+const InnerWrapper = styled.div`
   flex-shrink: 0;
   width: 50rem;
   max-width: calc(100vw - ${GEL_SPACING_QUAD});
 `;
 
-const LiveRadioContainer = ({ idAttr, externalId, id }) => {
+const AudioPlayer = ({ externalId: _masterBrand, id: assetId, idAttr }) => {
+  const { liveRadioOverrides, lang, translations, service } = useContext(
+    ServiceContext,
+  );
+  const masterBrand = getMasterBrand(_masterBrand, liveRadioOverrides);
   const { isAmp, platform } = useContext(RequestContext);
-  const { liveRadio, lang, translations, service } = useContext(ServiceContext);
   const location = useLocation();
   const isValidPlatform = ['amp', 'canonical'].includes(platform);
 
-  if (!isValidPlatform || !externalId || !id) return null;
+  if (!isValidPlatform || !masterBrand || !assetId) return null;
 
-  const serviceId = pathOr(
-    externalId,
-    ['externalIdOverrides', externalId],
-    liveRadio,
-  );
-
-  const embedSource = getEmbedUrl({
-    mediaId: `${serviceId}/${id}/${lang}`,
+  const embedUrl = getEmbedUrl({
+    mediaId: `${masterBrand}/${assetId}/${lang}`,
     type: 'media',
     isAmp,
     queryString: location.search,
```

Comment:
I know we didn't really cover this in the example mapping session, and I think we might need to open this out to the larger team (possibly a question for @ryanmccombe?) - the data in the test environment for On Demand Radio is not great, and the decision has been made to use live data instead of test data.  I am wondering whether we need to append `renderer_env=live` here, because we will need to play the episode on `polling.bbc.com` rather than `polling.test.bbc.com`

Reply:
It can be handled in Morph or Mozart - it's how live radio works without the `renderer_env` override.

But it's probably easier to do it in Simorgh.  Considering this "always use live data" constraint is just a developer experience enhancement for us, it would be better to have it in our code base than another team's

But I don't know if it's in scope for this PR - the page itself hasn't implemented this behavior yet for the ARES data, so we still have to add the `renderer_env` override anyway, which will also filter through to the embed URL

Reply:
I'll maybe need a little more context on this to fully understand the problem. One for next standup?